.github/workflows/pull_request_ubuntu_build.yml

```diff
@@ -62,7 +62,6 @@ jobs:
           export DISPLAY=':99.0'
           /usr/bin/Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
           ./gradlew build -Pnative.test --max-workers=2 --scan --no-daemon 
-          find ~/.gradle/caches/ -name "*.lock" -type f -delete
```

Comment:
Do we need this change?

Reply:
yes. there are some caching issues in the PR build for all `6.x`, `7.x`, `8.x` and master workflows. This includes the fix for `8.x`. 
Duplication of the PR https://github.com/ballerina-platform/ballerina-lang/pull/41346